path/glob.ts

```diff
@@ -1,28 +1,26 @@
 // Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
 // This module is browser compatible.
 
-import { isWindows, OSType } from "./_os.ts";
-import type { GlobOptions } from "./_common/glob_to_reg_exp.ts";
-import {
-  globToRegExp as posixGlobToRegExp,
-  joinGlobs as posixJoinGlobs,
-  normalizeGlob as posixNormalizeGlob,
-} from "./posix/glob.ts";
-import {
-  globToRegExp as windowsGlobToRegExp,
-  joinGlobs as windowsJoinGlobs,
-  normalizeGlob as windowsNormalizeGlob,
-} from "./windows/glob.ts";
+export type { GlobOptions } from "./_common/glob_to_reg_exp.ts";
+export type { GlobToRegExpOptions } from "./glob_to_regexp.ts";
 
-export { isGlob } from "./_common/is_glob.ts";
+import { globToRegExp as _globToRegExp } from "./glob_to_regexp.ts";
+import { normalizeGlob as _normalizeGlob } from "./normalize_glob.ts";
+import { joinGlobs as _joinGlobs } from "./join_globs.ts";
 
-export type { GlobOptions };
+import { isGlob as _isGlob } from "./is_glob.ts";
 
-export type GlobToRegExpOptions = GlobOptions & {
-  os?: OSType;
-};
+/**
+ * @deprecated (will be removed in 0.215.0) Import from "std/path/windows/is_glob.ts"
+ *
+ * Test whether the given string is a glob
+ */
+export const isGlob = _isGlob;
```

Comment:
```suggestion
export {
  /**
   * @deprecated (will be removed in 0.215.0) Import from "std/path/windows/is_glob.ts"
   *
   * Test whether the given string is a glob
   */
  isGlob,
} from "./is_glob.ts";
```
I think this approach is cleaner. Ditto for the other instances.

Reply:
Yep 👍🏾